ratis-logservice/src/main/java/org/apache/ratis/logservice/server/LogStateMachine.java

```diff
@@ -246,14 +250,21 @@ public StateMachineStorage getStateMachineStorage() {
    * @param msg message
    * @return reply message
    */
+  private CompletableFuture<Message> processGetSizeRequest(LogServiceRequestProto proto) {
+    GetLogLengthRequestProto msgProto = proto.getLengthQuery();
```

Comment:
Is it correct? Should not it be photo.getSizeRequest()?

Reply:
Hrm, that does look wrong :). Let me double check.

Reply:
Yup, definitely wrong. Wasn't causing a test failure because we don't need to pull anything off of the request object. Fixed it nonetheless.